### ISSUE TITLE
minikube start

### ISSUE THREAD

**Issue by peterzandbergen** [kubernetes/minikube#10236]
Hi,
the amd64 deb file seems to be the ARM version:

```
peza@CBS-R90WNRJN:~/DevProjects/rsatest$ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
sudo dpkg -i minikube_latest_amd64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 19.6M  100 19.6M    0     0  22.4M      0 --:--:-- --:--:-- --:--:-- 22.4M
peza@CBS-R90WNRJN:~/DevProjects/rsatest$ sudo dpkg -i minikube_latest_amd64.deb
dpkg: error processing archive minikube_latest_amd64.deb (--install):
 package architecture (arm64) does not match system (amd64)
Errors were encountered while processing:
 minikube_latest_amd64.deb
```

**Comment by afbjorklund** [comment]
Duplicate, #10224

Please try the binary, until the .deb has been updated

```
curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
sudo install minikube-linux-amd64 /usr/local/bin/minikube
```

It is the same binary, either way. Or well, supposed to be...

**Comment by peterzandbergen** [comment]
Hi Anders,

Thanks for your reply. I will give it a try.

**Comment by afbjorklund** [comment]
Should be fixed again in 1.17.1

**Comment by vvitad** [comment]
Hi. Same problem in 2023.
